packages/gafl-webapp-service/src/locales/cy.json

```diff
@@ -211,28 +205,28 @@
   "explination": "!",
   "facebook_bulletpoint_link": "facebook.com/environmentagency",
   "facebook_bulletpoint": "Facebook: ",
-  "feedback_link_prefix": "Bydd eich",
+  "feedback_link_prefix": "Your",
   "feedback_link_suffix_alt": " to help us improve the service.",
-  "feedback_link_suffix": "yn ein helpu i wella’r gwasanaeth",
-  "feedback_link_text": "adborth (yn agor mewn tab newydd)",
-  "footer_accessibility": "Datganiad Hygyrchedd",
-  "footer_cookies": "Cwcis",
-  "footer_copyright": "Mae’r holl gynnwys ar gael o dan Drwydded Llywodraeth Agored v3.0, ac eithrio lle nodir yn wahanol",
-  "footer_ea_helpline_charges_link": "Darganfyddwch fwy am daliadau galwadau",
-  "footer_ea_helpline_opening_hours": "Dydd Llun i ddydd Gwener, 8am i 6pm",
-  "footer_ea_helpline": "Llinell Gymorth Asiantaeth yr Amgylchedd:",
-  "footer_privacy_policy": "Polisi Preifatrwydd",
-  "footer_refund_policy": "Polisi Ad-daliad",
+  "feedback_link_suffix": "will help us improve the service",
+  "feedback_link_text": "feedback (opens in new tab)",
+  "footer_accessibility": "Accessibility Statement",
+  "footer_cookies": "Cookies",
+  "footer_copyright": "All content is available under the Open Government Licence v3.0, except where otherwise stated",
+  "footer_ea_helpline_charges_link": "Find out about call charges",
+  "footer_ea_helpline_opening_hours": "Monday to Friday, 8am to 6pm",
+  "footer_ea_helpline": "Environment Agency Helpline:",
+  "footer_privacy_policy": "Privacy Policy",
+  "footer_refund_policy": "Refund Policy",
   "free": "free",
   "gov_uk": " - GOV.UK",
-  "header_service_name": "Mynnwch drwydded pysgota gwialen",
+  "header_service_name": "Get a rod fishing licence",
   "identification": "Identification",
   "identify_body_protect_info": "To find your licence details we first need to identify you. This helps us protect your personal information.",
   "identify_error_empty_postcode": "You did not enter your postcode",
   "identify_error_empty": "Enter the last six digits of your licence number",
   "identify_error_enter_bday_max": "Your date of birth must be in the past",
   "identify_error_enter_bday_min": "Your date of birth is too long ago",
-  "identify_error_enter_bday": "Nodwch eich dyddiad geni a chynnwys y diwrnod, y mis a’r flwyddyn",
+  "identify_error_enter_bday": "Nodwch eich dyddiad geni a chynnwys y diwrnod, y mis a'r flwyddyn",
```

Comment:
Which apostrophe should we be using? I remember there was a ticket to change it from `'` to `’`